### ISSUE TITLE
Add support for enabling serial port

### ISSUE THREAD

**Issue by xobs** [fedberry/fedberry-config#28]
The Raspberry Pi 3 breaks the serial port unless `enable_uart=1` is added to /boot/config.txt, at which point it shows up as `/dev/ttyS0`.

I've heard that `raspian-config` includes an option to do this, but haven't looked.

**Comment by agrez** [comment]
I remember looking at this a while ago (when I first got a RPi3) but as the issue impacts bluetooth support and was more involved than first meets the eye I didn't add it as an option. I figured if people really want to have hardware UART available on GPIO pins 14 and 15 (and then either disable bluetooth or switch it to using software UART), then they should also be capable of manually editing their config.txt file.

**Comment by xobs** [comment]
Should `#enable_uart=1` be added to `/boot/config.txt` then?  Most other configuration options are present as comment stubs, but not that one.

**Comment by agrez** [comment]
Yes, that's another option. When I get some spare time, I will take another look at adding this as a config option in fedberry-config.
